lib/validators/liveValidator.ts

```diff
@@ -231,15 +248,15 @@ export class LiveValidator {
 
               // Whitelist lookups: Look up knownTitleToResourceProviders
               // Putting the provider namespace onto operation for future use
-              if (title && (Constants.knownTitleToResourceProviders as any)[title]) {
-                operation.provider = (Constants.knownTitleToResourceProviders as any)[title]
+              if (title && (C.knownTitleToResourceProviders as any)[title]) {
+                operation.provider = (C.knownTitleToResourceProviders as any)[title]
               }
```

Comment:
@amzavery@microsoft.com  this list includes only Microsoft.Resources. is this logic still necessary and if so should we incldue other RP's ?